File: src/main/java/com/sudosoo/takeiteasy/dto/comment/CreateCommentRequestDto.java
package com.sudosoo.takeiteasy.dto.comment;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor
public class CreateCommentRequestDto {
    @NotNull(message = "멤버아이디를 입력해 주세요.")
    private Long memberId;
    @NotNull(message = "게시글아이디를 입력해 주세요.")
    private Long postId;
    @NotNull(message = "내용을 입력해 주세요.")
    @Size(max = 500, message = "내용은 500자를 넘을 수 없습니다.")
    private String content;
}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/application/dto/comment/CreateCommentRequestDto.kt
package com.sudosoo.takeItEasy.application.dto.comment

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size
class CreateCommentRequestDto (
    @NotBlank(message = "멤버아이디를 입력해 주세요.")
    val memberId: Long,
    @NotBlank(message = "게시글아이디를 입력해 주세요.")
    val postId: Long,
    @NotBlank(message = "내용을 입력해 주세요.")
    @Size(max = 500, message = "내용은 500자를 넘을 수 없습니다.")
    val content: String
)
